nf_core/lint.py

```diff
@@ -524,8 +524,8 @@ def check_actions_branch_protection(self):
                 if has_name and has_if and has_run:
                     self.passed.append((5, "GitHub Actions 'branch' workflow checks that forks don't submit PRs to master: '{}'".format(fn)))
                     break
-            else:
-                self.failed.append((5, "Couldn't find GitHub Actions 'branch' workflow step to check that forks don't submit PRs to master: '{}'".format(fn)))
+        else:
```

Comment:
Hi Olga, the else here is part of a for/else loop (@ewels added it, I did not even know that for/else loops exist) so the indentation should be fine like it was before